src/codeu/chat/client/core/InterestsContext.java

```diff
@@ -0,0 +1,27 @@
+package codeu.chat.client.core;
+
+import java.util.Collection;
+
+import codeu.chat.common.BasicController;
+import codeu.chat.common.BasicView;
+import codeu.chat.common.ConversationHeader;
+import codeu.chat.common.Interests;
+import codeu.chat.util.Uuid;
+
+
+public final class InterestsContext {
+    public final Interests interests;
```

Comment:
I think @lysanderp41 mentioned this in the video chat but interestscontext.java might not even be needed.